### DISCUSSION TITLE
Angular 17 and PrimeNG 17 Support

### DISCUSSION THREAD

**Discussion by DIanAguilar28** [primefaces/community#714] [PrimeNG Templates] [answered]
Hi,

I just want to ask if you have an updated Atlantis theme that supports both PrimeNG 17 and Angular 17. 

Thanks.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

We're updating the templates including Atlantis to the v17. We're planning to deliver updated templates this week. Worst case scenario, it'll be available on the upcoming release day next week Wednesday.

**Reply by DIanAguilar28** [reply]
hi @cetincakiroglu was v17 released?

**Comment by goseivietnam** [comment]
Hi,
Do we have v17 on tomorrow :)?
